.github/workflows/backend.yml

```diff
@@ -154,24 +154,6 @@ jobs:
           app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
           private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
 
-      - name: Setup Python ${{ matrix.python-version }}
-        uses: ./.github/actions/setup-python
-        with:
-          python-version: ${{ matrix.python-version }}
-          # We just need requirements-dev, this lets us pull in less files.
-          cache-files-hash: ${{ hashFiles('requirements-dev.txt') }}
-
-      - uses: actions/cache@v3
-        with:
-          path: ~/.cache/pre-commit
-          key: cache-epoch-1|${{ env.pythonLocation }}|${{ hashFiles('.pre-commit-config.yaml') }}
```

Comment:
Are we loosing the pre-commit caching perf gain by removing this?

Reply:
yeah we want this still

Reply:
Oh, whoops. Good catch.